src/app/(general)/api/borrow/borrow.js

```diff
@@ -50,6 +66,170 @@ function borrow_book(params) {
         year_level: params['year_level'],
         section: params['section']
       }
+    },
+
+  }
+}
+
+
+function borrow_bookFaculty(params) {
+  return {
+    date: params['date'],
+    borrow_date: new Date().toISOString(),
+    return_date: null,
+    status: 'borrow',
+    type: params['type'],
+    user_type: params['user_type'],
+    bookRequests: {
+      create: {
+        book: {
+          connect: {
+            id: params['materialID']
+          }
+        }
+      }
+    },
+    user_faculty: {
```

Comment:
This can be optimized and just attribute a property to the object so we can exclude this 
```
{
    date: params['date'],
    borrow_date: new Date().toISOString(),
    return_date: null,
    status: 'borrow',
    type: params['type'],
    user_type: params['user_type'],
    bookRequests: {
      create: {
        book: {
          connect: {
            id: params['materialID']
          }
        }
      }
    },
```

but for now okay na muna to